interfaces/policy/policy.go

```diff
@@ -59,6 +59,34 @@ func (connc *ConnectCandidate) slotSnapType() snap.Type {
 	return connc.Slot.Snap.Type
 }
 
+func (connc *ConnectCandidate) plugSnapID() string {
+	if connc.PlugSnapDeclaration != nil {
+		return connc.PlugSnapDeclaration.SnapID()
+	}
+	return "" // unset
```

Comment:
The code is already saying that it's unset. Perhaps you mean something more specific here?